src/components/Gallery/Gallery.tsx

```diff
@@ -0,0 +1,22 @@
+import './Gallery.scss'
+
+type GalleryProps = { galleryArray: string[] }
+
+const Gallery = ( { galleryArray }: GalleryProps) => {
+
+const gallery = galleryArray ?? ["Sorry! Image not found."]
+
+  return (
+
+    <div className="gallery"> 
+      {gallery.map((image: string) => {
+        return <div className="gallery__images"><img className="gallery__images--image" src={image} alt="Image of a product from maker" /></div>
```

Comment:
check the bem on this one feels strange to have gallery__images--image could it not just be 

```
gallery__images {
//some css
 &--image{
//some css
}
}
```